staff/carlota-vicens/spotify-app/versions/v1.1.0/scripts/web-components-1.0.0.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+/**
+ * Web Components lib
+ * 
+ * @author manuelbarzi
```

Comment:
You should change the authory of the document!

Reply:
tu també! @bcasasus